Changes to config pages for airgapped client

donnyyung opened the pull request.

thallgren commented on docs/v2.3/reference/cluster-config.md:
````diff
@@ -118,9 +118,35 @@ run this command to generate the Cluster ID:
+  images:
+    agentImage: <privateRegistry>/<agentImage>
+  ```
+
````
I don't remember, hence this question: Will the client reinstall the traffic-manager when the value of `agentImage` changes? If not, I think it should be mentioned that the traffic-manager must be reinstalled.
Also, I think it's worth mentioning that changing this value is global and affects all telepresence users in the cluster.

donnyyung replied:
Nah so the `agentImage` will override agents if they already exist and the value is different, but it won't force a deploy of the traffic-manager (same for the webhoookAgentImage and webhookRegistry). 

the `registry` flag will deploy a new traffic-manager if it is different than the one that is in the cluster (And it wasn't deployed via helm chart).

I've updated the docs to make all this more clear, but let me know if it's not.

nbkrause commented on docs/pre-release/reference/cluster-config.md:
````diff
@@ -118,9 +118,35 @@ run this command to generate the Cluster ID:
+Users will now be able to use selective intercepts with the
 `--preview-url=false` flag (since use of preview URLs requires a connection to Ambassador Cloud).
 
+If using Helm to install the server-side components, see the chart's [README](https://github.com/telepresenceio/telepresence/tree/release/v2/charts/telepresence) to learn how to configure the image registry and license secret.
+
+### Have users of the cli add the following to their config.yml
+Telepresence attempts to auto-detect if the cluster is air-gapped,
+but you can add this to your config.yml to ensure the client behaves
+as if it was in an air-gapped environment.
+  ```
+  cloud:
+    skipLogin: true
+  ```
+
+Reminder: To use selective intercepts, which normally require a login, you
+must have a license in your cluster and specify which agentImage should be installed,
+by also adding the following to your config.yml:
+  ```
+  images:
+    agentImage: <privateRegistry>/<agentImage>
+  ```
+
````
Maybe nitpicky but I wonder if this would be better under an `airgapped` section in the Laptop-Side Configuration document since this doc is titled like it should only deal with cluster-side config.

donnyyung replied:
Yeah good point, I've moved it as you suggested, and added a link here so it tells the operator that there is some client-side configuration that is recommended.

donnyyung commented:
Since I addressed all the comments here and we want to get 2.3.5 docs in front of ppl like Bjorn for final review, I'm going to merge this into the `rel/2.3.5` branch, but if there are things you'd still like me to change, please leave a comment here: https://github.com/telepresenceio/telepresence.io/pull/25
